src/bool.cr

```diff
@@ -46,6 +46,11 @@ struct Bool
     hasher.bool(self)
   end
 
+  # Returns `1` for `true` and `0` for `false`.
+  def to_i
```

Comment:
You renamed the PR but didn't push new code?

Reply:
That's weird, I thought I did. Pushed it now.

Reply:
Check again.

Reply:
Wow, that was stupid of me. I amended the commit but didn't actually re-add my changes. Sorry about that.